CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Consider migrating to `stream-chat-android-ui-components` or `stream-chat-androi
   - Moshi 1.13.0
   - OkHttp 4.9.3
   - and other, see [#2771](https://github.com/GetStream/stream-chat-android/pull/2771) for more details
+- Markdown support is moved into a standalone module `stream-chat-android-markdown-transformer`.
```

Comment:
This is a breaking change. We should add more information on how to use it - for example - we can add a link to the docs
```suggestion
- 🚨 Breaking change: Markdown support is moved into a standalone module `stream-chat-android-markdown-transformer`.
```